R/prepExtendedComment.R

```diff
@@ -0,0 +1,49 @@
+#' prepExtendedComment
+#'
+#' Helper function condense metadata information into
+#' an extended comment entry
+#'
+#' @md
+#' @param x list containing the metadata to be condensed
+#' @param type output type, e.g. "TauTotal"
+#' @param warn boolean indicating whether warnings should be triggered
+#' if entries are missing, or not.
+#' @param n the number of generations to go back
```

Comment:
What is a generation here? I feel this arg's documentation needs more clarification